cmd/kubeadm/app/preflight/checks.go

```diff
@@ -437,9 +437,12 @@ func (hst HTTPProxyCheck) Name() string {
 // Check validates http connectivity type, direct or via proxy.
 func (hst HTTPProxyCheck) Check() (warnings, errorList []error) {
 	klog.V(1).Infoln("validating if the connectivity type is via proxy or direct")
-	u := (&url.URL{Scheme: hst.Proto, Host: hst.Host}).String()
+	u := &url.URL{Scheme: hst.Proto, Host: hst.Host}
+	if utilsnet.IsIPv6String(hst.Host) {
```

Comment:
> cmd/kubeadm/app/preflight/checks.go:441:5: undefined: utilsnet

package `utilsnet` is not imported here.

Reply:
Thanks! Fixed it.